code/src/Components/newThoughtInput.css

```diff
@@ -0,0 +1,69 @@
+.new-thought-input__form {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  background: #efefef;
+  border: 1px solid #000;
+  margin: 20px;
+  box-shadow: 5px 5px #000;
+  width: 300px;
+}
+
+.new-thought-input__text-area {
+  resize: none;
+  width: 100%;
+  margin: 5px 0;
+  font-size: 16px;
+}
+
+::placeholder {
+  font-size: 12px;
+}
+
+.new-thought-input__button-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.new-thought-input__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #000;
+  background-color: #ffadad;
+  font-weight: 700;
+  font-size: 14px;
+  height: 40px;
+  border-radius: 20px;
+}
+
+.new-thought-input__button:hover {
+  cursor: pointer;
+  border: 1px solid gray;
+}
+
+.new-thought-input__button:disabled {
+  opacity: 0.6;
+  cursor: not-allowed;
```

Comment:
This is awesome! Such a simple detail that makes totally difference. I will definitely use this in the future.